orchestrator/orchestratorAutomation.ps1

```diff
@@ -60,13 +78,13 @@ $STATIC_NODES_STRING='
     <add key="AiFabric.Logs" value="false" />
     <add key="AiFabric.ModuleEnabled" value="true" />
     <add key="AiFabric.FeatureEnabledByDefault" value="true" />
+    <add key="AiFabric.MLPackagingInstructionsUrl" value="something" />
+    <add key="AiFabric.MLServiceUrl" value="https://{{hostName}}" />
+    <add key="AiFabric.MLSkillUrl" value="https://{{hostName}}/ai-deployer" />
+    <add key="AiFabric.MLPackageUrl" value="https://{{hostName}}/ai-pkgmanager" />
+    <add key="AiFabric.MLLogUrl" value="https://{{hostName}}/ai-helper" />
+    <add key="AiFabric.MLTrainUrl" value="https://{{hostName}}/ai-trainer" />

```

Comment:
ai-appmanager missing here?

Reply:
am not sure whether it's being used by activity, if it's required by activity , can you plz raise an another PR with the same entry @aniket202